native/packages/react-native-bpk-component-banner-alert/src/style/index.android.js

```diff
@@ -0,0 +1,65 @@
+import {
+  colorGray50,
+  spacingBase,
+  spacingMd,
+  spacingSm,
+  spacingLg,
+  lineHeightXs,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import commonStyle from './style.common';
+
+const androidStyles = Object.assign({},
+  commonStyle, {
+    outerContainerSuccess: {
+      borderColor: colorGray50,
+    },
+    outerContainerWarn: {
+      borderColor: colorGray50,
+    },
+    outerContainerError: {
+      borderColor: colorGray50,
+    },
+    outerContainerNeutral: {
+      borderColor: colorGray50,
+    },
+    outerContainer: Object.assign({},
+      commonStyle.outerContainer,
+      {
+        backgroundColor: colorGray50,
+        borderWidth: 0,
+      }),
+    bannerContainerPadded: Object.assign({},
+      commonStyle.bannerContainerPadded,
+      {
+        backgroundColor: colorGray50,
+        paddingBottom: 0,
+        paddingLeft: 0,
+        paddingRight: 0,
+      }),
+    bannerContainerPaddedDismissable: Object.assign({},
+      commonStyle.bannerContainerPaddedDismissable,
+      {
+        paddingRight: spacingMd,
+      }),
+    bannerContainer: Object.assign({},
+      commonStyle.bannerContainer,
+      {
+        backgroundColor: colorGray50,
+        minHeight: 0,
+      }),
+    text: Object.assign({},
+      commonStyle.text,
+      {
+        paddingTop: spacingBase,
+        paddingBottom: spacingBase,
+        paddingRight: spacingLg,
+        lineHeight: lineHeightXs,
+      }),
+    icon: {
+      paddingVertical: spacingBase + (spacingBase / spacingBase),
```

Comment:
the padding is the same as the text, however the icon render with `1dp` padding on top, might be due to font scaling so I'm adding that 1dp, to avoid magic number I'm dividing `spacingBase` for itself, if there is any other better idea please let me know